napari/utils/_tests/test_notification_manager.py

```diff
@@ -135,7 +135,7 @@ def _raise():
         exception_thread.join(timeout=DEFAULT_TIMEOUT_SECS)
 
         try:
-            raise ValueError("a")
+            raise ValueError("a")  # noqa TRY301
```

Comment:
As this seems to be a relatively common pattern in tests, should/can we skip this rule for tests?

Reply:
So are you saying we should not skip the rule in tests?

Reply:
I do not know. I have added an explanation for full context, but maybe ignoring it globally for ests could be a good idea.

Reply:
Ok. I have ignored TRY301 in all test files